src/components/UserManagement/__test__/UserSearchPanel.test.js

```diff
@@ -2,27 +2,75 @@ import React from 'react';
 import { screen, render, fireEvent } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import UserSearchPanel from '../UserSearchPanel';
+import thunk from 'redux-thunk';
+import configureStore from 'redux-mock-store';
+import { renderWithProvider } from '../../../__tests__/utils';
+const mockStore = configureStore([thunk]);
+const nonJaeAccountMock = {
+  _id: '5edf141c78f1380017b829a6',
+  isAdmin: true,
+  user: {
+    expiryTimestamp: '2023-08-22T22:51:06.544Z',
+    iat: 1597272666,
+    userid: '5edf141c78f1380017b829a6',
+    role: 'Volunteer',
+    email: 'non_jae@hgn.net',
+    permissions:["getUserProfiles"]
+  },
+  firstName: 'Non',
+  lastName: 'Petterson',
+  role: {
+    roles:[]
+  },
+  weeklycommittedHours: 10,
+  email: 'non_jae@hgn.net'
+}
 
-describe('user search panel', () => {
+const ownerAccountMock = {
+  _id: '5edf141c78f1380017b829a6',
+  isAdmin: true,
+  user: {
+    expiryTimestamp: '2023-08-22T22:51:06.544Z',
+    iat: 1597272666,
+    userid: '5edf141c78f1380017b829a6',
+    role: 'Owner',
+    email: 'devadmin@hgn.net'
+  },
+  firstName: 'Dev',
+  lastName: 'Admin',
+  weeklycommittedHours: 10,
+  email: 'devadmin@hgn.net',
+  role: {
+    roles:[]
+  }
+}
+
+describe('user search panel not admin', () => {
   let onNewUserClick;
   let onSearch;
   let onActiveFilter;
+  let store;
   beforeEach(() => {
+    store = mockStore({
+      auth: nonJaeAccountMock,
+      userProfile: nonJaeAccountMock,
+      role: nonJaeAccountMock.role
```

Comment:
```suggestion
      role: rolesMock.role
```
Pretty sure the `role` here is actually an array of roles with their respective permissions used to determine which permissions roles have. I think it's a bit misleading that it's singular (the structure is actually `role: roles: {dictionary of roles[list of permissions for each role]}`)

This isn't the user's role, but about the permissions of roles in general.